src/main/java/roomescape/web/controller/ReservationController.java

```diff
@@ -31,15 +33,15 @@ public ResponseEntity<List<ReservationResponse>> getReservations() {
 	}
 
 	@PostMapping
-	public ResponseEntity<ReservationResponse> create(@RequestBody ReservationRequest request) {
-		ReservationRequest.validateReservation(request);
-		return ResponseEntity.ok().body(this.reservationService.create(request));
+	public ResponseEntity<ReservationResponse> create(@RequestBody ReservationRequest request,
+			LoginMember loginMember) {
+		return new ResponseEntity<>(this.reservationService.create(request, loginMember), HttpStatus.CREATED);
 	}
 
 	@DeleteMapping("/{id}")
 	public ResponseEntity<Void> cancel(@PathVariable("id") long id) {
 		this.reservationService.cancel(id);
-		return ResponseEntity.ok().build();
+		return new ResponseEntity<>(HttpStatus.NO_CONTENT);
```

Comment:
```suggestion
        return ResponseEntity.noContent().build();
```

여기도 `ResponseEntity` 쪽에 존재하는 메서드를 활용해볼 수 있을 것 같구요!
(다른 클래스도 확인해주세요)